src/components/HeadlineUnderlined.tsx

```diff
@@ -36,7 +38,9 @@ const HeadlineUnderlined: React.FC<Props> = ({
         id={id}
         className={`${
           light ? "text-white" : "text-tolo-black"
-        } relative z-10 font-semibold leading-none font-aktiv ${large ? "text-5xl" : "text-2xl"}`}
+        } relative z-10 font-semibold leading-none font-aktiv ${
+          large ? "text-5xl" : "text-[28px]"
```

Comment:
What's the reason for the change from `text-2xl` to `text-[28px]`?

Reply:
I've changed to text-[28px] because text-2xl left part of the underline vector to the right. And text-3xl went over the vector's length.
When I've inspected the figma design, the size of the heading font is 28px which uses the entire length of the vector. I thought it was a visually good solution. 
What would have been your approach? 🤔

Reply:
This is a component that is reused for all these underlined headlines. If you look at them in Figma, they're all defined to be 48px which is `text-5xl` in Tailwind. So in this case, we can be very precise and stick with Figma by keeping it at `text-5xl`. So please revert this change and then let's take a look at it together and where it doesn't look good.